doc/releases/reef.rst

```diff
@@ -6,6 +6,438 @@ Reef is the 18th stable release of Ceph. It is named after the reef squid
 (Sepioteuthis).
 
 
+v18.2.4 Reef
+============
+
+This is the fourth backport release in the Reef series. We recommend that all users update to this release.
+
+An early build of this release was accidentally exposed and packaged as 18.2.3 by the Debian project in April.
+That 18.2.3 release should not be used. The official release was re-tagged as v18.2.4 to avoid
+further confusion.
+
+v18.2.4 container images, now based on CentOS 9, may be incompatible on older kernels (e.g., Ubuntu 18.04) due
```

Comment:
incompatible *with* older kernels

Also, Ubuntu 18.04 isn't a kernel.

(e.g., the 4.15 kernel shipped with Ubuntu 18.04)

Ideally we would identify the threshold kernel version.